app/views/pages/item-detail/item-details.js

```diff
@@ -0,0 +1,268 @@
+import React from 'react/addons';
+import _ from 'lodash';
+import helpers from '../../components/helpers';
+import TagsInput from '../../components/tags-input';
+import ItemDetailMixin from './detail-mixin';
+import ItemTitle from './item-title';
+import ItemAttachments from './item-attachments';
+import ProductActions from '../../../actions/product-actions';
+import {State} from 'react-router';
+import ScoreMap from '../../../lib/score-map';
+import STATUS_MAP from '../../../lib/statuses-map';
+const INVERTED_STATUS_MAP = _.zipObject(_.values(STATUS_MAP), _.keys(STATUS_MAP))
+
+var ItemDetails = React.createClass({
+
+  mixins: [State, ItemDetailMixin],
+
+  propTypes: {
```

Comment:
Actually now I remember - the component renders before the item has been fetched in the instance of hitting the item detail url instead of navigating to it via an item card

It ends up just being noisy in the console